benefits/core/admin.py

```diff
@@ -26,6 +27,33 @@
     admin.site.register(model)
 
 
+class EligibilityTypeForm(forms.ModelForm):
```

Comment:
I was hoping this could be done without a custom admin form? 

I think implementing [`clean()`](https://docs.djangoproject.com/en/5.0/ref/models/instances/#django.db.models.Model.clean) in the model class should be enough?

Reply:
Oh I see now that it could be done that way too and that we don't need the custom admin form. I'll put this back in draft and move the logic to be done on the model